bootstrap-app/src/cmr/bootstrap/services/dispatch/core_async_dispatch.clj

```diff
@@ -0,0 +1,143 @@
+(ns cmr.bootstrap.services.dispatch.core-async-dispatch
+  "Provides methods to insert migration requets on the appropriate channels."
+  (:require
+    [clojure.core.async :as async :refer [>!]]
+    [cmr.bootstrap.services.dispatch.dispatch-protocol :as dispatch-protocol]
```

Comment:
I don't believe you want to be doing anything protocol related here ... this becomes more clear once you re-org the code into implementation namespaces.